examples/capi-templates-images.yml

```diff
@@ -0,0 +1,105 @@
+###############################################################################
+# Configuration of Glance software images.
+
+ubuntu-focal-kube-v1_25_11:
+  name: "ubuntu-focal-kube-v1.25.11"
+  type: qcow2
+  image_url: "https://object.arcus.openstack.hpc.cam.ac.uk/swift/v1/AUTH_f0dc9cb312144d0aa44037c9149d2513/azimuth-images/ubuntu-focal-kube-v1.25.11-230712-0939.qcow2"
+  is_public: True
+  properties:
+    os_hidden: "False"
+    os_distro: "ubuntu"
+    os_version: "20.04"
+    kube_version: "v1.25.11"
+
+ubuntu-focal-kube-v1_26_6:
+  name: "ubuntu-focal-kube-v1.26.6"
+  type: qcow2
+  image_url: "https://object.arcus.openstack.hpc.cam.ac.uk/swift/v1/AUTH_f0dc9cb312144d0aa44037c9149d2513/azimuth-images/ubuntu-focal-kube-v1.26.6-230712-1010.qcow2"
+  is_public: True
+  properties:
+    os_hidden: "False"
+    os_distro: "ubuntu"
+    os_version: "20.04"
+    kube_version: "v1.26.6"
+
+ubuntu-focal-kube-v1_27_3:
+  name: "ubuntu-focal-kube-v1.27.3"
+  type: qcow2
+  image_url: "https://object.arcus.openstack.hpc.cam.ac.uk/swift/v1/AUTH_f0dc9cb312144d0aa44037c9149d2513/azimuth-images/ubuntu-focal-kube-v1.27.3-230712-1021.qcow2"
+  is_public: True
+  properties:
+    os_hidden: "False"
+    os_distro: "ubuntu"
+    os_version: "20.04"
+    kube_version: "v1.27.3"
+
+# List of Glance images. Format is as required by the stackhpc.os-images role.
+openstack_images:
+  - "{{ ubuntu-focal-kube-v1_25_11 }}"
+  - "{{ ubuntu-focal-kube-v1_26_6 }}"
+  - "{{ ubuntu-focal-kube-v1_27_3 }}"
+
+###############################################################################
+# Configuration of Magnum container clusters.
+
+kube-v1_25_11:
+  labels: "kube_tag=v1.25.11"
+  external-network-id: "external"
+  master-flavor: "m1.small"
+  flavor: "m1.small"
```

Comment:
Lets make a variable for these flavors?

Like magnum_default_controller flavor, etc.